x-pack/plugins/lens/public/datatable_visualization/components/toolbar.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiFlexGroup, EuiFormRow, EuiSwitch } from '@elastic/eui';
+import { ToolbarPopover } from '../../shared_components';
+import { VisualizationToolbarProps } from '../../types';
+import { DatatableVisualizationState } from '../visualization';
+
+export function DataTableToolbar(props: VisualizationToolbarProps<DatatableVisualizationState>) {
+  const { state, setState } = props;
+
+  return (
+    <EuiFlexGroup gutterSize="none" justifyContent="spaceBetween" responsive={false}>
+      <ToolbarPopover
```

Comment:
`EuiFlexGroup` should have `EuiFlexItem` as child

Reply:
ToolbarPopover is component which include EuiFlexItem on the first level